Mage.Sets/src/mage/cards/g/GiftOfTheViper.java

```diff
@@ -0,0 +1,42 @@
+package mage.cards.g;
+
+import java.util.UUID;
+
+import mage.abilities.effects.common.UntapTargetEffect;
+import mage.abilities.effects.common.counter.AddCountersTargetEffect;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.counters.CounterType;
+import mage.target.common.TargetCreaturePermanent;
+
+/**
+ *
+ * @author grimreap124
+ */
+public final class GiftOfTheViper extends CardImpl {
+
+    public GiftOfTheViper(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[] { CardType.INSTANT }, "{G}");
+
+        // Put a +1/+1 counter, a reach counter, and a deathtouch counter on target creature. Untap it.
+        this.getSpellAbility().addEffect(new AddCountersTargetEffect(CounterType.P1P1.createInstance())
+                .setText("Put a +1/+1 counter").concatBy(","));
```

Comment:
`.setText("Put a +1/+1 counter")` is enough, `concatBy` is not used for the first effect in text generation.